configsnap.help2man

```diff
@@ -0,0 +1,25 @@
+[Files]
```

Comment:
This may be done as a separate PR but if we are adding a man page, we should document all functions of the application here ie: at lease what is printed in configsnap --help but preferably more in depth.